views/js/pciCreator/dev/audioRecordingInteraction/runtime/js/recorder.js

```diff
@@ -200,8 +199,35 @@ define([
                         initAnalyser(stream);
 
                         setState(recorder, recorderStates.IDLE);
+                    })
+                    .catch(function(err) {
+                        alert(err);
                     });
             },
+
+            /**
+             * Create the Audio context
+             * @returns {AudioContext}
+             */
+            initAudioContext: function initAudioContext() {
+                var context = window.audioContext;
+
+                if (context) {
+                    audioContext = context;
+                } else {
+                    var AudioContext = window.AudioContext || window.webkitAudioContext;
```

Comment:
It's against the `vars-on-top` ESLint rule.

Reply:
Also I don't prefer to have an `audioContext` and an `AudioContext` variable. The variable name should reflect the purpose of it. What do you think?